contents/blog/hog-story.md

```diff
@@ -0,0 +1,206 @@
+
+But this lack of shared certainty did not close the path forward. It was enough that Marius was confident enough to keep making progress.
+
+Again, PostHog’s approach to management is to let the people closest to a problem make the decisions on how to solve it. As part of this process, we’re encouraged to solicit opinions on from others who may be affected or who have experience in this space.
+
+But crucially, the handbook [says](https://posthog.com/handbook/company/communication#requests-for-comment-rfcs) this about requests for comment:
+
+> You don't need to reach full agreement to decide, particularly if the decision is reversible. Instead, it should be when the decision maker has considered the feedback and is confident in their decision.
+>
+> As the decision maker, you should use your judgment as to which comments you want to respond fully to. It's fine to politely decline a question if you think it's not required for the decision being made.
+
+This policy is organizational insurance against the sort of committee thinking and filibustering that de-rails bold decisions. Instead, PostHog explicitly empowers the prime mover of a new direction to take action regardless of opinion.
+ 
+Personal judgment matters, of course. You could use this policy self-destructively, shredding important relationships and social fabric.
+ 
+But in the case of creating our languages, it gave Marius enough room to lay the foundations for the next generation of PostHog technology, even as not everyone shared his certainty.
+ 
+Now, thanks to HogQL and Hog, PostHog gained:
+ 
+ - Much easier to maintain insights in product analytics. These are core features, and the improved developer experience has compound returns each time we onboard someone to that team
+ - Data warehouse: the ability to collect, absorb and join data not just from PostHog, but from other services as well. This impacts the value of the company as a whole: we can be a complete solution in this space
+ - Data pipelines: completely customizable logic for packaging and transmitting PostHog data, in real time, to external platforms
+ - Dashboard reporting which is user-customizable down to the query-level
+ - The ability to let our customers, in short, create their own business logic for data collection and reporting, without the overhead of maintaining an entire bespoke reporting platform themselves
+
+These are consequential returns for the investment of building these languages. We now have rich soil for the next generation of PostHog products, just as we’re hitting the point of our trajectory where we need to hire and build more than we ever have.
+
+How many companies failed to carve out resources to invest in the future, caught in a constant [doom loop of tech debt and artificial deadlines](deadlines piece)? I’ve seen it plenty. I bet you have too.
+
+# The business case for imagination
```

Comment:
Section also feels super long and could be combined with the final few paragraphs of the previous section.